package.json

```diff
@@ -15,12 +15,13 @@
   ],
   "scripts": {
     "test": "lerna run test",
-    "postinstall": "run-s build:elements build:tokens",
+    "postinstall": "npm run build",
     "prep-release": "lerna version --conventional-commits --yes",
     "start": "concurrently --kill-others --names \"storybook,elements,tokens\" -c \"bgCyan,bgMagenta,bgYellow\" \"yarn start:react\" \"yarn start:elements\" \"yarn start:tokens\"",
     "start:react": "cd packages/components/react-web && yarn start",
     "start:elements": "cd packages/elements && yarn start",
     "start:tokens": "cd packages/components/tokens && yarn start",
+    "build": "lerna run build",
```

Comment:
I had a problem after running `lerna bootstrap` where `postinstall` would fail

`lerna run build` basically does the same thing - it runs the `build` script for every package that has such

Reply:
Great idea—I should have thought of this! 😜